app/js/results/smaaResults.html

```diff
@@ -7,11 +7,45 @@
 </div>
 <div ng-if="state.results">
   <div class="grid-x">
+    <div class="cell large-3">
+      Scenario:
+      <br><br>
+    </div>
+    <div class="cell large-9">
+      <select id="scenario-selector" class="scenario-select ng-pristine ng-valid ng-not-empty ng-touched"
```

Comment:
a bunch of these ng-* properties are set automatically by angular and should not be in the code